frontend/app.js

```diff
@@ -476,6 +485,24 @@
         });
     });
 
+    /**
+     * Function to intercept the access of pages that should be displayed only on mobile screens.
+     * @param {service} $transitions - Service of transitions states
+     */
+    app.run(function mobileInterceptor($transitions, STATES) {
+        const permitted_routes = [
+            STATES.CREATE_EVENT
+        ];
+
+        $transitions.onSuccess({
+            to: (state) => {
+                return !Utils.isMobileScreen(475) && _.includes(permitted_routes, state.name);
+            }
+        }, (transition) => {
+            transition.router.stateService.transitionTo(STATES.HOME);
```

Comment:
create a page that show "Esta página só está disponível para mobile" and a button to go to home state